README.md

```diff
@@ -1,2 +1,12 @@
-# deal-thief
-Robbery
+# deal-thief  
+Deal Thief is a website that will allow users to find the best hotel deals all in one place. The ability to find the best current prices by city and date.  
+
+## URL
+https://dealthief.herokuapp.com
+
+## Developers  
+Mike Harrison, Steven Than, Victor Benavente
+
+
+Woked with Cris and looked at Twitter Tunes for help with Mock Testing
+github.com/icarrera/twitter-tunes/
```

Comment:
You might wish to include more information in your README.  Things like how to set up the project and get it running, how to contribute if you're a developer, so on.